crates/sui-indexer-alt/src/handlers/sum_obj_types.rs

```diff
@@ -99,8 +99,7 @@ impl Processor for SumObjTypes {
                                     Owner::ObjectOwner(_) => StoredOwnerKind::Object,
                                     Owner::Shared { .. } => StoredOwnerKind::Shared,
                                     Owner::Immutable => StoredOwnerKind::Immutable,
-                                    // TODO: Implement support for ConsensusV2 objects.
-                                    Owner::ConsensusV2 { .. } => todo!(),
+                                    Owner::ConsensusV2 { .. } => StoredOwnerKind::ConsensusV2,
```

Comment:
I thought that for objects we wanted to treat consensus-owned objects as address-owned if they the authenticator maps to a single address, which means no new kind here. (Then there was a future question of how we handle party objects which we kind of punted on, but the options varied from treating them like shared objects, to having some separate category for jointly-owned objects).

Reply:
updated as per huddle